cmake/clang-cxx-dev-tools.cmake

```diff
@@ -0,0 +1,17 @@
+# Additional targets to perform clang-format/clang-tidy
+# Get all project files
+file(GLOB_RECURSE
+        ALL_CXX_SOURCE_FILES
+        *.[chi]pp *.[chi]xx *.cc *.hh *.ii *.[CHI]
+        )
+
+# Adding clang-format target if executable is found
+find_program(CLANG_FORMAT "clang-format")
+if(CLANG_FORMAT)
+    add_custom_target(
+            clang-format
+            COMMAND "${CLANG_FORMAT}"
+            -i
+            ${ALL_CXX_SOURCE_FILES}
```

Comment:
Would be nice to add here as well: `-style="${PROJECT_SOURCE_DIR}/.clang-format"`

Reply:
>Invalid value for -style

I dunno